includes/abstracts/abstract-wc-rest-terms-controller.php

```diff
@@ -533,6 +533,14 @@ public function delete_item( $request ) {
 		}
 
 		$term = get_term( (int) $request['id'], $taxonomy );
+		// Get default category id.
+		$default_category_id = absint( get_option( 'default_product_cat', 0 ) );
+
+		// Prevent deleting the default product category
```

Comment:
Just update this line to contain the colon so that PHPCS passes and we are good to go.
```suggestion
		// Prevent deleting the default product category.
```